sipi/scripts/file_info.lua

```diff
@@ -97,12 +101,10 @@ local text_extensions = {
     "csv"
 }
 
+-- TODO should be tidied up!
```

Comment:
I have a general idea to add TODO with the suffix - and identifier of person who added it, so for example I use it like that `TODO-mpro: comment`. With this it's easy to look for your own TODO's and one doesn't need to look for the blame to check who add that.

Reply:
WIth current state of API and our creating new issues policy I'm not sure if this going to work like that. On top of that - we are still in learning process and this is IMO the best way to leave notes - otherwise can be easily forgotten.

Reply:
I agree that it shouldn't be forbidden to leave TODO notes in the code. And I like the idea of signing them with a suffix.  
But I also very much think we should avoid creating TODO comments that remain untouched for 5 years, like it's been done before. (In fact, we should at some point get rid of all the old TODOs!)

For this case, I simply removed it because it's not within the scope of the PR.

Reply:
I'm sure we can work out some less extremal form between avoiding TODOs and leaving them for 5+ years unresolved :)

Reply:
I hope so too :)